### DISCUSSION TITLE
Lona Application Example: lona-infobeamer

### DISCUSSION THREAD

**Discussion by SmithChart** [lona-web-org/lona#235] [Show and tell]
This year we have (again) conducted our hacker camping ( https://hackenopenair.de/en.html ). We wanted deploy some displays that show current information, system stats, price lists, etc.

On many German hacker conferences https://info-beamer.com/ is used. But we were not happy with the feemium pricing model. So we decided to write such application ourselfs. And so we have written lona-infobeamer :grin: 
It's based on lona for the web frontend and used django for the dabase.
The tool allows you to upload images and add urls. Images and URLs are then added to queues. Queues can be assigned to outputs (eg. a display). Then images and urls are shown in a fixed 10s interval.
The code is here: https://gitli.stratum0.org/chrissi/lona-infobeamer

During the event we've added a lona-script based server that provided dashboards for some on-site systems (e.g. https://engelsystem.de/ and the worshop schedule from our wiki). The code is here: https://gitli.stratum0.org/HOA/infobeamer-engelsystem

During the event we've had 5 displays running. The display were powered by some rather old snapdragon-based single-board-computers. Rendering images (JPG, PNG) and simple webpages worked like a charm. But displaying complex web-pages (like a Grafana-Dashboard) did not work. The SBCs are simply too slow to render the pages in time.

Maybe having some reference projects helps others working with lona.

**Comment by fscherf** [comment]
@SmithChart: thats absolutely great! Thanks for that! We should add the link, and some Screenshots to the examples page, or we create a section named "Projects using Lona" or something like that
